llama.cpp

```diff
@@ -4644,6 +4636,8 @@ static void llm_load_vocab(
             vocab.special_pad_id  = -1;
             vocab.special_cls_id  = -1;
             vocab.special_mask_id = -1;
+        } else {
+            throw std::runtime_error(format("unknown tokenizer: '%s'", tokenizer_model.c_str()));
```

Comment:
This seems like a good change. A default tokenizer feels like a bad idea, and throwing an exception to require users specify an explicit tokenizer feels like the right choice. 👍